### ISSUE TITLE
Invitation links to private channels don't work on windows

### ISSUE THREAD

**Issue by danielcybulka** [ZbayApp/zbay#131]
Version: 1.0.68-dev-test-windows-2
Platform: Windows 10 x64
zcashd: v2.1.2

Opening the link with the invitation and choosing to open it with Zbay causes:
1. When Zbay is opened - JavaScript error occurs and after dismissing it channel is not added.
![Friend link windows](https://user-images.githubusercontent.com/67123884/85120459-8e58b880-b223-11ea-9b41-ef20a1a2ae0b.PNG)

2. When Zbay is closed - zbay opens, syncs but no channel is added.

In other words, it's impossible to join private channel on windows.

**Comment by holmesworcester** [comment]
note: only blocks windows release

**Comment by jkdev601** [comment]
Fix in #166
